### ISSUE TITLE
Unable to launch corteza-server cannot decode settings for "record.attachments.max-size": json: cannot unmarshal string into Go value of type uint 

### ISSUE THREAD

**Issue by Toasterson** [cortezaproject/corteza-server#106]
**Describe the bug**
The Max attachment and page attachment types are uint (See https://github.com/cortezaproject/corteza-server/blob/25a2f531456a79b2448ed11365b3e7b650effc68/compose/types/settings.go#L25) but the db field is JSON string. That is an impossible configuration and leads to the following error. `cannot decode settings for "record.attachments.max-size": json: cannot unmarshal string into Go value of type uint` As it is possible to set the settings in the admin area, it is importent that they work even though they are not yet implemented. At least they should not prevent the server from starting up when they are set.

**To Reproduce**
- Go to Compose settings in the Corteza Admin Area.
- Set either `Page attachments->Max Size` or `Record attachments->Max Size`
- restart server. (Or wait for it to crash)
- server fails to start with the error `cannot decode settings for "record.attachments.max-size": json: cannot unmarshal string into Go value of type uint`

**Expected behavior**
Server starts

**Screenshots**
![Screenshot of Settings area with error](https://user-images.githubusercontent.com/3026297/91025738-124f6800-e5fa-11ea-89ae-dc6b64708c68.png)

**Comment by tjerman** [comment]
Hi, thank you for the detailed report!

We've stumbled upon this a bit back and will be resolved in the next release (2020.9).
In the meantime, please correct this issue manually, directly in the database (table `compose_settings`) by deleting the setting or changing it to a numeric value.

**Comment by Toasterson** [comment]
Yes already did. It cannot be changed to numeric as far as i can tell. Or at least I did not manage to write that as proper SQL. I managed to delete the entries and then the server started again.

**Comment by Toasterson** [comment]
SQL statement for others 
```SQL
DELETE FROM compose_settings WHERE name='record.attachments.max-size' OR name='page.attachments.max-size';
```

**Comment by darh** [comment]
fixed in 04711ce4
